tracer/src/Datadog.Trace/IAST/Propagation/StringModuleImpl.cs

```diff
@@ -22,6 +22,120 @@ internal static class StringModuleImpl
         return value == null ? null : taintedObjects.Get(value);
     }
 
+    public static object? TaintIfInputIsTainted(object input, object result)
```

Comment:
I find this name slight awkward, maybe PropagateTaint would be better.

Reply:
Fixed!